ftplugin/racket.vim

```diff
@@ -7,7 +7,7 @@ if exists("b:did_ftplugin")
 endif
 let b:did_ftplugin = 1
 
-setl iskeyword+=#,%,^
+setl iskeyword+=#,%,^,-
```

Comment:
This appears to be invalid. If not mistaken, this will cause an error:

```
E474: Invalid argument: iskeyword+=#,%,^,-
```

The caret must be the last item. You might need to change it to:

```
setl iskeyword+=#,%,-,^
```

But I'm not sure how you can guarantee that the original `iskeyword` you are appending to does not contain `^` at the end.

Reply:
@cwfoo thanks for reviewing.